readme.md

```diff
@@ -294,6 +294,7 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [bpftrace](https://bpftrace.org/) - A tool for tracing with its own high-level tracing language. It is flexible enough to be envisioned as a Linux replacement for DTrace and SystemTap.
   - [bpftrace Cheat Sheet](https://www.brendangregg.com/BPF/bpftrace-cheat-sheet.html) - Summary and cheat sheet for programming in bpftrace. Contains information about syntax, probe types, variables and functions.
 - [kubectl trace](https://github.com/iovisor/kubectl-trace) - A kubectl plug-in for executing bpftrace programs in a Kubernetes cluster.
+- [inspektor-gadget](https://github.com/inspektor-gadget/inspektor-gadget) - A collection of of eBPF-based tools (or gadgets) to debug and inspect Kubernetes resources and applications.
```

Comment:
If you could just remove the duplicate `of`, please.

```suggestion
- [inspektor-gadget](https://github.com/inspektor-gadget/inspektor-gadget) - A collection of eBPF-based tools to debug and inspect Kubernetes resources and applications.
```

Nit: The notion of "gadget" is probably unnecessary here.